pkg/parser/parser.peg

```diff
@@ -1500,12 +1486,15 @@ UnquotedAttributeChar <- [^\r\n"=\],] { // excludes comma
     return types.NewStringElement(string(c.text))
 }
 
-// rule used after document attribute substitution, to discover links 
+// rule used after document attribute substitution, to discover links
+// TODO This is unused
```

Comment:
it's used in https://github.com/bytesparadise/libasciidoc/blob/master/pkg/parser/document_processing_apply_substitutions.go#L133, as the inline links can be "discovered" after substituting attributes

Reply:
Huh... I thought it came up unused in my editor... I'll remove it.

Reply:
it's not used elsewhere in the grammar, but it is explicitly used as an entrypoint when parsing the line with substituted attributes (and the entrypoint is also defined in the `make generate` targets)

Reply:
Fixed, regenerated the parser, and rebased on top of master.